fuse_publishers/CMakeLists.txt

```diff
@@ -49,12 +51,28 @@ target_link_libraries(${PROJECT_NAME}
   ${catkin_LIBRARIES}
 )
 
+add_executable(graph_dot_writer_node
+  src/graph_dot_writer.cpp
+  #src/graph_dot_writer_node.cpp
```

Comment:
I still need to split the `graph_dot_writer.cpp` into a writer itself and the node. That way I could even write a unit test.